core/src/main/java/org/infinispan/persistence/manager/TransactionalPersistenceManager.java

```diff
@@ -0,0 +1,21 @@
+package org.infinispan.persistence.manager;
+
+import org.infinispan.marshall.core.MarshalledEntry;
+
+import javax.transaction.Transaction;
+import java.util.List;
+
+/**
+ * @author Ryan Emerson
+ */
+public interface TransactionalPersistenceManager extends PersistenceManager {
```

Comment:
I am debating maybe we should either just add these all onto the same interface or not tie the interfaces together and just have the implementer implement both.  The latter would allow us to split into 2 implementations as well if that helps anywhere, just wondering.